### PR TITLE
No longer truncates messgae, now splits to new columns

### PR DESCRIPTION
**Pull request by DarianHole**
Sorry for the new branch, I was having an issue updating the other branch.

- New code now splits 'qc_message' into more than one column
- New output is "Biohansel_Bionumerics" from "output.csv"

### CODE REVIEW

File: tools/biohansel_bionumeric_converter/bionumeric_converter.py [hunk 1]
```diff
@@ -24,30 +24,26 @@ def main():
...
-    for count in df.index:
-        message = str(df.at[count, 'qc_message'])
-        if len(message) > 150:
-            results = message.find('|')
-            new_message = "Truncated after first '|' : " + message[0:results]
-            df['qc_message'] = df['qc_message'].replace(message, new_message)
+    for i, row in df.iterrows():
+        message = row['qc_message']
+        try:
+            if len(message) > 150:
```
Comment (Takadonet) [inline tools/biohansel_bionumeric_converter/bionumeric_converter.py]: Allow it to split up the message across as many columns as needed.

File: tools/biohansel_bionumeric_converter/bionumeric_convert.xml [hunk 2]
```diff
@@ -1,38 +1,41 @@
...
+        **Outputs:**
+
+        - A .CSV file or a collection of .CSV files called "*Output*" that can be renamed and downloaded as required.
+
+        **Specific modifications done to the data**
 
         1. Converts all commas in the output to "/"
-        2. Shortens BioHansel qc_messages if they are over 150 characters
-        3. Converts the .tab file to a .csv file
+        2. Splits Biohansel qc_message column into multiple columns if the message is longer than 150 characters (up to 450 characters at the moment)
```
Comment (Takadonet) [inline tools/biohansel_bionumeric_converter/bionumeric_convert.xml]: remove "up to" comment when changes are fixed below.

File: tools/biohansel_bionumeric_converter/bionumeric_converter.py [hunk 3]
```diff
@@ -24,30 +24,29 @@ def main():
...
-            new_message = "Truncated after first '|' : " + message[0:results]
-            df['qc_message'] = df['qc_message'].replace(message, new_message)
+    for i, row in df.iterrows():
+        message = row['qc_message']
+        try:
+            message_list = list(splittingstrings(message, 150))
+            df.at[i, 'qc_message'] = message_list[0]
+            for val in range(1, len(message_list)):
+                df.at[i, 'qc_message_{}'.format(val)] = message_list[val]
+        except TypeError:
```
Comment (Takadonet) [inline tools/biohansel_bionumeric_converter/bionumeric_converter.py]: why do you have a TypeError with only pass line?

Comment (DarianHole) [reply tools/biohansel_bionumeric_converter/bionumeric_converter.py]: You get a TypeError (object of type 'float' has no len()) when there is nothing in the QC column. This allows it to keep the original qc_message and not break

Comment (DarianHole) [reply tools/biohansel_bionumeric_converter/bionumeric_converter.py]: Ah nevermind I have a better workaround

**Pull request merged**